static/katex.less

```diff
@@ -553,4 +553,7 @@
         }
     }
 
+    .relpos {
+        position: relative;
+    }
```

Comment:
Maybe using a negative margin will work better with the non-combining character since it'll have a non-zero width.  Regardless, adding a prefix is probably a good idea.